modules/kmm-day0-day1-installation.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/kmm-preflight-validation.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-day0-day1-installation_{context}"]
+= Day 0 and Day 1 kmod installation
```

Comment:
@qbarrand Should the title be changed to "Day 0 through Day 2"?

Reply:
Yes please.

Reply:
Changed title.